src/shims-vue.d.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="vite/client" />
```

Comment:
is this supposed to be here?

Reply:
Yes, it's necessary for things like `import.meta.env`